src/commands/aws/role.ts

```diff
@@ -46,10 +46,10 @@ export const role = (yargs: yargs.Argv<{ account: string | undefined }>) =>
       .demandCommand(1)
   );
 
-const isOktaSamlConfig = (
-  config: AwsItemConfig
-): config is AwsItemConfig & { uidLocation: AwsOktaSamlUidLocation } =>
-  config.uidLocation?.id === "okta_saml_sso";
+const isFederatedLogin = (
+  config: AwsItem
+): config is AwsItem & { login: AwsFederatedLogin } =>
+  config.login?.type === "federated";
```

Comment:
Should this be the equivalent stricter check that verifies all the way down to `login.provider.method == saml` (and `okta`, etc.)?

Reply:
not necessary I think; when we add more complications we can update this